crates/optimism/evm/src/l1.rs

```diff
@@ -370,8 +369,8 @@ where
     // If the canyon hardfork is active at the current timestamp, and it was not active at the
     // previous block timestamp (heuristically, block time is not perfectly constant at 2s), and the
     // chain is an optimism chain, then we need to force-deploy the create2 deployer contract.
-    if chain_spec.is_fork_active_at_timestamp(OpHardfork::Canyon, timestamp) &&
-        !chain_spec.is_fork_active_at_timestamp(OpHardfork::Canyon, timestamp.saturating_sub(2))
+    if chain_spec.is_canyon_active_at_timestamp(timestamp) &&
+        !chain_spec.is_cancun_active_at_timestamp(timestamp.saturating_sub(2))
```

Comment:
```suggestion
        !chain_spec.is_cancun_active_at_timestamp(timestamp.saturating_sub(2))
```

Is this cancun in the second one or canyon?